server/graphql/v1/queries.js

```diff
@@ -666,14 +666,15 @@ const queries = {
           throw new Error('Collective not found');
         }
         const getCollectiveIds = () => {
-          // if is host, we get all the expenses across all the hosted collectives
+          // if is host, we get all the expenses across all the hosted collectives that are active
           if (args.includeHostedCollectives) {
-            return models.Member.findAll({
+            return models.Collective.findAll({
+              attributes: ['id'],
               where: {
-                MemberCollectiveId: collective.id,
-                role: 'HOST',
+                HostCollectiveId: collective.id,
+                isActive: true,
               },
-            }).map(member => member.CollectiveId);
+            }).map(c => c.id);
```

Comment:
```suggestion
            }).map(collective => collective.id);
```

It's better with an explicit variable name.

Reply:
I actually had written "collective" in full at first and then renamed it to avoid confusion with a variable in the scope above. And since it's a very short lived variable, I thought it was ok to use just `c`.